enhancements/update/change-management-and-maintenance-schedules.md

```diff
@@ -0,0 +1,1220 @@
+With any cluster update, there is risk --  software is changing and even subtle differences in behavior can cause
+issues given an unlucky combination of factors. Teams responsible for cluster operations are familiar with these
+risks and owe it to their stakeholders to minimize them where possible.
+
+The current, fully self-managed, update process makes one obvious risk mitigation strategy
+a relatively advanced strategy to employ: only updating the control-plane and leaving worker-nodes as-is.
+It is possible by pausing machine config pools, but this is certainly not an intuitive step for users. Farther back 
+in OpenShift 4's history, the strategy was not even safe to perform since it could lead to worker-node 
+certificates to expiring. 
+
+By separating the control-plane and worker-node updates into two separate steps, we provide a clear
+and intuitive method of deferring worker-node updates: not initiating them. Leaving this to the user's
+discretion, within safe skew-bounds, gives them the flexibility to make the right choices for their
+unique circumstances.
+
+It should also be noted that Service Delivery does not permit customers to directly modify machine config
+pools. This means that the existing machine config pool based pause is not directly available. It is 
+not exposed via OCM either. This enhancement seeks to create high level abstractions supporting the 
+separation of control-plane and worker-nodes that will be straight-forward and intuitive options
+to expose through OCM.
+
+#### Enhancing Operational Control
+The preceding section delved deeply into a motivation for Change Management / Maintenance Schedules based on our desire to 
+separate control-plane and worker-node updates without increasing operational burden on end-users. However,
+Change Management, by providing control over exactly when updates & material changes to nodes in
+the cluster can be initiated, provide value irrespective of this strategic direction. The benefit of
+controlling exactly when changes are applied to critical systems is universally appreciated in enterprise 
+software. 
+
+Since these are such well established principles, I will summarize the motivation as helping
+OpenShift meet industry standard expectations with respect to limiting potentially disruptive change 
+outside well planned time windows. 
+
+It could be argued that rigorous and time sensitive management of OpenShift cluster API resources could prevent
+unplanned material changes, but Change Management / Maintenance Schedules introduce higher level, platform native, and more 
+intuitive guard rails. For example, consider the common pattern of a gitops configured OpenShift cluster.
+If a user wants to introduce a change to a MachineConfig, it is simple to merge a change to the
+resource without appreciating the fact that it will trigger a rolling reboot of nodes in the cluster.
+
+Trying to merge this change at a particular time of day and/or trying to pause and unpause a 

```

Comment:
+1 , in 4.16 (as TechPreview) MCO has introduced a user facing cluster level API using which admin can specify what action to take if change involve only certain change that admin thinks is safe. 
Not sure, how much this feature makes impact to this EP since it targets improving UX for customers who doesn't want to go through the pain of pause/unpause of pools.   Also, reboot will take priority if the overall MachinceConfig diff has changes that definitely require reboot like OS Update, kernelArgs change.

Reply:
https://github.com/openshift/enhancements/pull/1525 is a great EP and I think it melds well with this proposal. A `nodeDisruptionPolicy` which states a MachineConfiguration does not require a node reboot becomes a non-material change in this proposal that can roll out immediate (I realize that applying changes like this could be material in the true sense of the word, but the cluster admin is specifically telling us "do it anyway" and so would not be gated by the maintenance schedule).

Reply:
Adding a section about this in Special Handling and resolving this thread.